specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/examples/PotentialOutages_List.json

```diff
@@ -0,0 +1,62 @@
+{
+  "parameters": {
+    "api-version": "2021-01-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "nextLink": "string",
+        "value": [
+          {
+            "id": "/providers/Microsoft.ResourceHealth/potentialoutages/default",
```

Comment:
>"id": "/providers/Microsoft.ResourceHealth/potentialoutages/default" [](start = 12, length = 68)

Since this is an array, can you please add at least two items in the array? I am curious to see what is the id of a second resource.

Reply:
the id will be the same for the second item.

Reply:
This is actually changing, and is addressed in a different PR.